kahypar/application/kahypar.cc

```diff
@@ -41,7 +41,9 @@ int main(int argc, char* argv[]) {
 
   kahypar::Hypergraph hypergraph(
     kahypar::io::createHypergraphFromFile(context.partition.graph_filename,
-                                          context.partition.k));
+                                          context.partition.k,
+                                          VALIDATE_INPUT,
```

Comment:
Would it make sense to move `VALIDATE_INPUT` and `PROMOTE_WARNINGS_TO_ERRORS` into the `context`? That way, users could enable/disable this during runtime instead of having to recompile.

Reply:
I see. Thanks for the explanation. This makes sense.